frontend/src/App.tsx

```diff
@@ -28,11 +28,27 @@ const App: FC = () => {
   const dispatch = useDispatch<AppDispatch>()
   const isStandalone = useSelector(selectModeStandalone)
   const loading = useSelector(selectLoadingMode)
-  useEffect(() => {
+  const getMode = (attempts: number) => {
+    attempts++
     dispatch(getModeStandalone())
+      .unwrap()
+      .catch(() => {
+        new Promise((resolve) =>
+          setTimeout(resolve, 1000 * 2 ** attempts),
+        ).then(() => {
```

Comment:
@ReiHashimoto I think if we do it like this, when the server fails, it will take a very long time to make api calls

Reply:
@sanglevinh Thank you for your review.
I replaced it to use constant waiting time.
Does it look fine?

Reply:
@ReiHashimoto Yes, I feel like this is quite reasonable

Reply:
@sanglevinh Thank you.
Can I merge this PR to your `feature/mode_standalone` branch?

Reply:
@ReiHashimoto yes, I removed the 
if (res.data === undefined) {
    return getModeStandaloneApi()
  }
you can merge it